precompiles/erc20/integration_test.go

```diff
@@ -209,11 +209,12 @@ var _ = Describe("ERC20 Extension -", func() {
 					},
 				)
 
-				Expect(res.GasUsed > expGasUsedLowerBound).To(BeTrue(), "expected different gas used")
-				Expect(res.GasUsed < expGasUsedUpperBound).To(BeTrue(), "expected different gas used")
+				Expect(res.GasUsed > expGasUsedLowerBound).To(BeTrue(), "expected gas used to be more than %d; got: %d", expGasUsedLowerBound, res.GasUsed)
+				Expect(res.GasUsed < expGasUsedUpperBound).To(BeTrue(), "expected gas used to be lower than %d; got: %d", expGasUsedUpperBound, res.GasUsed)
 			},
-				// FIXME: The gas used on the precompile is much higher than on the EVM
-				Entry(" - direct call", directCall, int64(3_021_000), int64(3_022_000)),
+				// FIXME: Interestingly, on these tests the gas used is not in the same range, but on the specific gas consumption tests it's shown that it fits quite good
+				// The gas values consumed here are in line with the ones consumed by the ERC20 Solidity contracts on the first call, but on follow up calls it will be cheaper.
+				Entry(" - direct call", directCall, int64(30_000), int64(31_000)),
```

Comment:
As can be seen in the gas tests results linked in the PR description, this is the amount for any calls except for the very first - that's why this value was chosen, even though it is lower than the values for the ERC20 Solidity calls here.